app/javascript/react/components/App.js

```diff
@@ -1,13 +1,17 @@
 import React from 'react'
 import RideIndex from './RideIndex'
+import RideShowContainer from './RideShowContainer'
+
 import { BrowserRouter, Route, Switch } from 'react-router-dom'
 
 export const App = (props) => {
   return (
     <div> 
       <BrowserRouter>
         <Switch>
-            <Route exact path="/rides" component={RideIndex} />
+          <Route exact path="/" component={RideIndex} />
+          <Route exact path="/rides" component={RideIndex} />
+          <Route exact path="/rides/:id" component={RideShowContainer} />
         </Switch>
      </BrowserRouter>
     </div>
```

Comment:
let's remove the `div` in this file as the BrowserRouter acts as our top level element